### ISSUE TITLE
FIXME error

### ISSUE THREAD

**Issue by francoisthire** [rlepigre/lambdapi#62]
The following file generates a FIXME error

```
def A : Type.
def eA : A -> Type.
a : A.
[] A --> eA a.

B : Type.
def eB : B -> Type.

[] eB a --> eA a.
```

**Comment by fblanqui** [comment]
14:03 ~ lambdapi zo.dk 
[zo.dk, 9:0-16] Error on command.
FIXME1.

**Comment by fblanqui** [comment]
When trying to type the LHS, we get the constraint A = B which, after normalization, becomes eA a = B. It is therefore unsatisfiable. This is perfect for SR: this means that the rule always satisfies SR since the LHS is never typable. But a warning should be output.

**Comment by rlepigre** [comment]
The first problem here is that a proper error message should be given.

https://github.com/rlepigre/lambdapi/blob/6069d2434dfe565fc81a9129c1296d0c63cdb953/src/solve.ml#L166

What should the message say here?

**Comment by fblanqui** [comment]
I think that this should be fixed by 8d249f47d9350ef99a1fbc10f6034b03568bb6cf.

**Comment by fblanqui** [comment]
The output is now:
15:06 ~/lambdapi-git ./lambdapi.native issue62.dk 
untypable LHS
Checked [issue62.dk]
